fuel/app/migrations/002_create_tours.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace Fuel\Migrations;
+
+class Create_tours
+{
+	public function up()
+	{
+		\DBUtil::create_table('tours', array(
+			'id' => array('constraint' => 11, 'type' => 'int', 'auto_increment' => true, 'unsigned' => true),
+			'code' => array('type' => 'text'),
+			'image' => array('type' => 'text'),
+			'title' => array('type' => 'text'),
+			'gross_min' => array('type' => 'text'),
+			'gross_max' => array('type' => 'text'),
```

Comment:
@Hung-Phan-Mulodo , pls replace code, name, country, country_name, city, city_name from text to int type.